cf/mixin/propertiesdatabounds.py

```diff
@@ -2079,6 +2079,41 @@ def file_locations(self):
 
         return out
 
+    @_inplace_enabled(default=False)
+    def filled(self, fill_value=None, bounds=True, inplace=False):
+        """Replace masked elements with a fill value.
+
+        .. versionadded:: NEXTVERSION
+
+        :Parameters:
+
+            fill_value: scalar, optional
+                The fill value. By default the fill returned by
+                `get_fill_value` is used, or if this is not set then
+                the netCDF default fill value for the data type is
+                used (as defined by `netCDF.fillvals`).
```

Comment:
As per the above comment:

```suggestion
                used (as defined by `cf.default_netCDF_fillvals`).
```